recipes/aiapy/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "aiapy" %}
+{% set version = "0.3.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aiapy-{{ version }}.tar.gz
+  sha256: 6937fc6ba3e931f2069baea4d5d2153122a5a6f0f6de9721f31733f12eecd18e
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - importlib_resources  # [py<37]
```

Comment:
It's safe to just have this dependency on `importlib_resources` without the selector. Yes, it's redundant with the standard library package on Python >= 3.8, but then you can make this `noarch:python`, which is a big win.